src/main/osd/osd_warnings.c

```diff
@@ -213,6 +213,13 @@ void renderOsdWarning(char *warningText, bool *blinking, uint8_t *displayAttr)
         return;
     }
 
+    if (osdWarnGetState(OSD_WARNING_LOAD) && (getArmingDisableFlags() & ARMING_DISABLED_LOAD)) {
+        tfp_sprintf(warningText, "CPU OVERLOAD");
+        *displayAttr = DISPLAYPORT_SEVERITY_CRITICAL;
+        *blinking = true;
+        return;
+    }
```

Comment:
Do we need `#if defined(USE_LATE_TASK_STATISTICS)` here also

Reply:
No, as something else could set that flag. Based on the testing below we might want to set this for CPU load above 90% for example, but I don't think that's required given this check.